changelogs/fragments/sysctl_fact_fix.yml

```diff
@@ -1,2 +0,0 @@
-bugfixes:
-  - setup/gather_facts will now warn and skip missing ``sysctl`` instead of being a fatal error.
```

Comment:
The missing sysctl bug is still fixed.
```yaml
bugfixes:
  - setup/gather_facts will now skip missing ``sysctl`` instead of being a fatal error.
```